src/cpp/rtps/builtin/discovery/endpoint/EDPSimpleListeners.h

```diff
@@ -33,6 +33,9 @@ namespace eprosima {
 namespace fastrtps {
 namespace rtps {
 
+using EndpointAddedCallback = std::function<
+    void (RTPSReader* reader, const CacheChange_t* const change)>;
```

Comment:
Should not the second `const` be removed?

Reply:
removed in [391527d](https://github.com/eProsima/Fast-DDS/pull/4240/commits/391527df3729a32aadbeb5815d70fd172648d916)